setup.sh

```diff
@@ -356,6 +364,11 @@ if [ $download == "yes" ]; then
     export PATH=~/.wks/bin:$PATH
 fi
 
+# On macOS, we only support the docker backend.
+if [ $(goos) == "darwin" ]; then
+    set_config_backend docker
+fi
+
```

Comment:
Since this ends up updating config.yaml we need to include that in the git add/commit/push - right now it is excluded and the setup fails.

Reply:
Ah yes, of course!